src/main/java/ca/uwaterloo/drinkmasterapi/feature/mqtt/service/MqttClientServiceImpl.java

```diff
@@ -0,0 +1,135 @@
+package ca.uwaterloo.drinkmasterapi.feature.mqtt.service;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.iot.client.AWSIotException;
+import com.amazonaws.services.iot.client.AWSIotMessage;
+import com.amazonaws.services.iot.client.AWSIotMqttClient;
+import com.amazonaws.services.iot.client.AWSIotQos;
+import com.amazonaws.services.securitytoken.AWSSecurityTokenService;
+import com.amazonaws.services.securitytoken.AWSSecurityTokenServiceClientBuilder;
+import com.amazonaws.services.securitytoken.model.Credentials;
+import com.amazonaws.services.securitytoken.model.GetSessionTokenRequest;
+import com.amazonaws.services.securitytoken.model.GetSessionTokenResult;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.UUID;
+
+@Service
+public class MqttClientServiceImpl implements MqttClientService {
+
+    private static final Logger logger = LoggerFactory.getLogger(MqttClientServiceImpl.class);
+
+    // examples
+    private static final String GDPR_TOPIC = "nvio/gdpr";
+    private static final String OTA_TOPIC = "nvio/ota";
+    private static final String LABEL_TOPIC = "nvio/label";
+    // Todo: create Topic in AWS IOT
+    private static final String POUR_TOPIC = "cmd/pour";
+    private static final String GETSTAT_TOPIC = "cmd/getstat";
+    private static final String STAT_TOPIC = "info/stat";
+    private static final String COMPLETE_TOPIC = "info/complete";
+    private static final String SETCFG_TOPIC = "cmd/setcfg";
```

Comment:
create a enum class for the constants, or set them as property variables of the system

Reply:
should i put this enum class under model?